README.md

```diff
@@ -55,7 +55,8 @@ curl -L https://raw.github.com/DamnWidget/VenGO/master/install.sh | bash
 Finally the command below will enable the `vengo` command in your system
 
 ```
-$ source $HOME/.VenGO/bin/vengo
+$ source $HOME/.VenGO/bin/vengo.sh
+$ source $HOME/.VenGO/bin/includes/*
```

Comment:
I will add this as a troubleshooting section as it is not needed to explicitly include the `includes` directory as they are implicitly included vi `vengo.sh`. For example I never had to do it in any of my Linux boxes or my OS X (I am using Gentoo and Manjaro with zsh by the way).

I tested it with `bash` as well as `csh`, `zsh` and `ash` and even there are scripts for `fish`, that the function is not defined in your system when you run `vengo_activate` is a problem of your SHELL/environment so I will add this as a workaround in case that the user encounters this problem.